packages/gatsby-theme-carbon/src/components/GifPlayer/GifPlayer.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import {
+  PlayOutline24,
+  PlayOutlineFilled24,
+  PauseOutline24,
+  PauseOutlineFilled24,
+} from '@carbon/icons-react';
+import styles from './GifPlayer.module.scss';
+
+const Pause = ({ hovering }) =>
+  hovering ? <PauseOutlineFilled24 /> : <PauseOutline24 />;
+
+const Play = ({ hovering }) =>
+  hovering ? <PlayOutlineFilled24 /> : <PlayOutline24 />;
+
+const ToggleIcon = ({ paused, hovering }) =>
+  paused ? <Play hovering={hovering} /> : <Pause hovering={hovering} />;
+
+const GifPlayer = ({ children, color, className, isInDialog }) => {
+  const [paused, setPaused] = useState(false);
+  const [hovering, setHovering] = useState(false);
+
+  const onClick = () => {
+    setPaused(!paused);
+  };
+
+  const controlsClassNames = classnames({
+    [styles.controls]: true,
+    [styles.dark]: color === 'dark',
+  });
+
+  const containerClassNames = classnames({
+    [styles.container]: true,
+    [className]: className,
+    [styles.gifInDialog]: isInDialog,
+  });
+
+  const staticImageClassNames = classnames({
+    [styles.imgHidden]: true,
+    [styles.imgDisplayed]: paused,
+  });
+
+  const gifClassNames = classnames({
+    [styles.gifDisplayed]: true,
+    [styles.gifHidden]: paused,
+  });
+
+  const childrenArray = React.Children.toArray(children);
+
+  return (
+    <div className={containerClassNames}>
+      <div className={gifClassNames} aria-hidden={paused ? 'true' : false}>
+        {childrenArray[0]}
+      </div>
+      <div
+        className={staticImageClassNames}
+        aria-hidden={paused ? false : 'true'}
+      >
+        {childrenArray[1]}
+      </div>
+      <button
+        type="button"
+        aria-label="play pause toggle"
```

Comment:
```suggestion
        aria-label="animation play pause toggle"
```
Just a hair more specific here and I think our intent is way clearer 👍